schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id AUTO_INCREMENT,
+    name varchar(100),
+    date_of_bith DATA,
```

Comment:
- [ ] There's a spelling error here. It is supposed to be `date_of_birth` NOT `date_of_bith`. Kindly look into it and fix.
- [ ] It also complains about the type `DATA`. It is supposed to the `date` not `data`. Kindly look into that and fix.


### Error
![spelling-error](https://github.com/theresetuyi/Vet-clinic-database/assets/26597432/e5f82384-64df-464d-95cf-f3048d8cdc8b)